Sources/SQLKit/SQLDialect.swift

```diff
@@ -9,6 +9,18 @@ public protocol SQLDialect {
     var supportsIfExists: Bool { get }
     var supportsAutoIncrement: Bool { get }
     var enumSyntax: SQLEnumSyntax { get }
+    var dropTriggerSupportsTableName: Bool { get }
```

Comment:
It would be nice to namespace these like `SQLEnumSyntax` is namespaced. That would make discoverability easier and organize things a bit better. 

```swift
protocol SQLDialect {
    ...
    var triggerSyntax: SQLTriggerSyntax { get }
}

struct SQLTriggerSyntax {
    struct Drop {
        var supportsTableName: Bool
        var supportsCascade: Bool
    }
    var drop: Drop
    struct Create {
        var requiresForEachRow: Bool
        var supportsBody: Bool
        ...
    }
    var create: Create
   ...
}
```

Reply:
I implemented this as OptionSet instead as that seemed more like the way to go.